packages/doc-site/stories/utilities/migrateDashboard.mdx

```diff
@@ -0,0 +1,55 @@
+import { Meta } from '@storybook/blocks';
+
+<Meta title="Utilities/migrateDashboard" />
+
+# migrateDashboard
+
+`migrateDashboard` is a function which allows you to easily migrate your Sitwise Monitor dashboards into applications developed using [Iot Application Kit](https://github.com/awslabs/iot-app-kit).
+
+## How to use
+
+To use `migrateDashboard` provide the `dashboardId` of your Sitewise Montitor dashboard and an object that contains the implementation of a `describeDashboard` API call. It returns
+a promise with the dashboard configuration of the now migrated AppKit dashboard which can be passed into the dashboard component.
+ 
+[Learn more about the dashboard](/docs/components-dashboard--docs) 
+
+## Example
+```
+    import { migrateDashboard } from '@iot-app-kit/dashboard';
+    import { IoTSiteWise } from '@aws-sdk/client-iotsitewise';
+    import { Dashboard } from '@iot-app-kit/dashboard';
+
+    const client = new IoTSiteWise({
+        credentials: awsCredentials,
+        region: awsRegion,
+    });
+
+    const MyDashboard = () => {
+        const options = {
+            params: {dashboardId: "sitewiseMonitorDashboardId"}
```

Comment:
Typo: API is parameters, not params, right?

Reply:
yep, forgot to update the docsite with change. done now!